### PR TITLE
Make sure we ignore all the errors in abort repair nemesis

### PR DESCRIPTION
**Pull request by fruch**
seem like some of the error appear in db logs after we remove the filters, wait a bit more time for them

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -826,6 +826,7 @@ def repair_streaming_exists():
 
             self.target_node.remoter.run('curl -X POST --header "Content-Type: application/json" --header "Accept: application/json" "http://127.0.0.1:10000/storage_service/force_terminate_repair"')
             thread1.join(timeout=120)
+            time.sleep(10)  # to make sure all failed logs/events, are ignored correctly
```
Comment (bentsi) [inline sdcm/nemesis.py]: how does this help?

**Comment by roydahan** [comment]
@bentsi, Is this solving #1163 ?

**Comment by bentsi** [comment]
da :) closed the issue (#1163)

**Comment by amoskong** [comment]
I can still see the abort db error even we already have the sleep from this PR.

@fruch @bentsi 

Job:
- https://jenkins.scylladb.com/view/scylla-3.1/job/scylla-3.1/view/scylla-3.1-test/job/longevity/job/longevity-100gb-4h/63/console

Logs:
- https://cloudius-jenkins-test.s3.amazonaws.com/0d3e8b0f-da18-4681-bac7-00bda43616e8/job_log.zip
- https://cloudius-jenkins-test.s3.amazonaws.com/0d3e8b0f-da18-4681-bac7-00bda43616e8/events_log.zip

**Comment by amoskong** [comment]
```
09-09 12:28:56,091 >>>Started random_disrupt_method abort_repair

12:29:06,466 f:nemesis.py      l:870  c:sdcm.nemesis         p:DEBUG > sdcm.nemesis.ChaosMonkey: Abort repair streaming by storage_service/force_terminate_repair API
12:29:17,423 f:nemesis.py      l:879  c:sdcm.nemesis         p:DEBUG > sdcm.nemesis.ChaosMonkey: Execute a complete repair for target node

 12:29:16.452: (DatabaseLogEvent Severity.CRITICAL): type=RUNTIME_ERROR regex=std::runtime_error line_number=11855 node=Node longevity-100gb-4h-3-1-db-node-0d3e8b0f-5 [54.197.38.120 | 172.30.0.178] (see
d: False)
 12:29:06+00:00  ip-172-30-0-178 !INFO    | scylla: [shard 6] repair - Got error in row level repair: std::runtime_error (repair id 1 is aborted on shard 6)
```

@fruch it seems the abort errors are still identified after sleeping 10 seconds.

**Comment by fruch** [comment]
look like the amount of error print was too much for the events queue to handle it in 10sec,
I'll dig further into why

**Comment by fruch** [comment]
seem like the logs are in delay of more the 10sec, which mean this fix isn't enough.
me @bentsi talked about possible solution
raise #1301 for handling the proper solution for this issue

**Pull request merged**